demos/appyx-interactions/android/src/main/kotlin/com/bumble/appyx/interactions/sample/SpotlightExperiment.kt

```diff
@@ -140,15 +149,47 @@ fun <InteractionTarget : Any> SpotlightUi(
             .padding(
                 horizontal = 64.dp,
                 vertical = 12.dp
-            ),
+            )
+            .semantics {
+                collectionInfo = CollectionInfo(1, spotlight.elements.value.all.size)
+            },
         element = { elementUiModel ->
+            val elementIndex = spotlight.elements.value.all.indexOf(elementUiModel.element)
+            val state = model.output.collectAsState()
+            val activeIndex = state.value.currentTargetState.activeIndex.toInt()
             Element(
                 color = color,
                 elementUiModel = elementUiModel,
-                contentDescription =
-                "${SPOTLIGHT_EXPERIMENT_TEST_HELPER}_${elementUiModel.element.id}",
                 modifier = Modifier
                     .fillMaxSize()
+                    .semantics(mergeDescendants = true) {
+                        customActions =
+                            if (elementIndex == activeIndex) {
+                                listOfNotNull(
+                                    CustomAccessibilityAction("Next") {
```

Comment:
Not localised.

Also can you use non-custom actions that are used in `androidx.compose.HorizontalPager`?

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/pager/Pager.kt;l=726?q=androidx.compose.foundation.pager&sq=&ss=androidx – `pageUp { }` & `pageDown { }`

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/pager/LazyLayoutSemanticState.kt;l=44 – `CollectionInfo` (you are already using it)

IDK how to make it focus the scrolled page (on your video it does not focus on it). probably somewhere in sources too. Please compare TalkBack behaviour to the default `HorizontalPager`.